docs/design/3_1_1_video_capture.tex

```diff
@@ -1 +1,29 @@
-TODO: Talk about camera choice, MIPI interface, camera libraries. Also, talk about alternatives we explored (i.e. analog video transmission/acquisition.
\ No newline at end of file
+% TODO: Talk about camera choice, MIPI interface, camera libraries. Also, talk about alternatives we explored (i.e. analog video transmission/acquisition.
+
+The video capture subsystem of the computing platform consists of a camera, its video transmission protocol to the processing core and video transmission to the base station. For this subsystem, the hardware used includes a CMOS camera, a Raspberry Pi device, and MIPI Camera Serial Interface protocol. Design description of the Raspberry Pi device is described in the next section.
+
+\subsubsection{Camera Choice}
+The camera used to capture video is the Raspberry Pi Camera Module. It supports 1080p, 720p and 640x480p video. It has a fixed focus lens. The camera will capture live video and send the video data to the Raspberry Pi, which will feed the data in different forms to both the machine learning model and the base station.
+
+% camera Choice
+The decision of the camera was influenced by the requirement of future replaceability. The project's goal is to create a system which provides both video acquisition and supplemental programmable hardware processing capabilities. The Raspberry Pi device contains different ports that will support different cameras with different performance.
+
+One of the limitations on the resolution and framerate of the camera is the latency of the machine learning implementation on the FPGA. The machine learning implementation is not the focus of this project and will not be optimized for performance and thus it will not be necessary to use a camera with high resolution and framerate.
+
+% {MIPI interface}
+\subsubsection{MIPI Interface}
+The MIPI camera serial interface (CSI) is a high-speed protocol primarily intended for point-to-point image and video transmission between cameras and host devices. 
+
+In the Requirements Specification, the camera is required to use a standard interface to allow for future replaceability. The MIPI protocol is a widely adopted protocol for image and video transmission. A hardware based MIPI core will be programmed into the programmable logic and will handle receiving of camera footage.
+
+% {Camera libraries or getting the video to a 'viewable' state}
+\subsubsection{Camera Libraries}
+Live video feed is a requirement for the base station. This is achieved by implementing a Python script on the Raspberry Pi for streaming the video to a webpage on the base station via Wi-FI. Justification for using Wi-FI is discussed in Section 3.1.5 - Communications. The script used to achieve this uses the Python package 'picamera,' which is a Python interface to the Raspberry Pi Camera. 
```

Comment:
Use the \ref command to link sections, rather than fully writing them out (just in case we move them around).

Also, why do we use the picamera library? Perhaps because it is the most widely supported library for this camera?